.gitignore

```diff
@@ -36,3 +36,6 @@ coverage
 # Synpress
 .cache-synpress
 test-results
+
+# Local scripts
+deploy.sh
```

Comment:
If this script is for deployment of contracts, can you push it as well?
I was thinking of adding one as well.

Reply:
Okay I'll include it, it's useful when you have to deploy everything